nncf/experimental/tensor/functions.py

```diff
@@ -218,7 +244,7 @@ def count_nonzero(a: TTensor, axis: Optional[Union[int, Tuple[int]]] = None) ->
 
 @functools.singledispatch
 @_tensor_guard
-def isempty(a: TTensor) -> TTensor:
+def isempty(a: TTensor) -> Tensor:
```

Comment:
Some comments about bool type.
Comparison operator can return `bool` too in case of comparing numbers, but return np.array or torch.tensor if compare arrays.
```python
>>> np.array([0,0]) < np.array([1,0])
array([ True, False])
>>> np.array(0) < np.array(1)
True
```

May be better to return Tensor in all case, include `isempty` and `allclose`? Or return bool too for comparison operator  in case of numbers?

Reply:
My comment was only about `isempty` function. I agree that `allclose` should return a Tensor. But I do not understand why isempty should return a Tensor.